fsharp-backend/tests/Tests/BwdServer.Tests.fs

```diff
@@ -194,29 +198,17 @@ let replaceByteStrings
     // bytes are added in reverse, so one more reverse needed
     result |> List.reverse |> List.toArray
 
-
-
-
-
-
-let t filename =
-  testTask $"Httpfiles: {filename}" {
-    let skip = String.startsWith "_" filename
-    let name =
-      let withoutPrefix = if skip then String.dropLeft 1 filename else filename
-      withoutPrefix |> String.dropRight 5 // ".test"
-
-    let filename = $"tests/httptestfiles/{filename}"
-    let! contents = System.IO.File.ReadAllBytesAsync filename
-
-    let test = parseTest contents
-    let canvasName =
-      match test.canvasName with
-      | Some name -> Exact name
-      | None -> Randomized $"bwdserver-{name}"
-
-    let! (meta : Canvas.Meta) = initializeTestCanvas canvasName
-
+/// Initializes and sets up a test canvas (handlers, secrets, etc.)
+let setupTestCanvas testName test : Task<Canvas.Meta> =
```

Comment:
Can you add type signatures for all parameters? (And if it's not in the coding guide can you add it there?)